code/controllers/configuration.dm

```diff
@@ -199,7 +199,6 @@
 	// Discord crap.
 	var/discord_url
 	var/discord_password
-	var/kill_phrase = "All your bases are belong to us."
```

Comment:
Holy shit this isn't even the quote

Reply:
Didn't matter much, it's a placeholder that gets replaced by a config defined value.

Reply:
I know, but imagine making a completely unnecessary reference to a meme and then getting the meme wrong